ibis/backends/tests/test_temporal.py

```diff
@@ -1337,6 +1337,13 @@ def test_interval_add_cast_column(backend, alltypes, df):
             ),
             "%Y%m%d",
             marks=[
+                pytest.mark.broken(
+                    [
+                        "datafusion",
+                    ],
+                    raises=com.UnsupportedArgumentError,
```

Comment:
I also noticed that other backends raise generic exceptions such as AttributeError (or ValueError). Would it be interesting to make those exceptions more specific, like `com.UnsupportedArgumentError`?